lib/openactive/validators/string_validator.rb

```diff
@@ -0,0 +1,26 @@
+module OpenActive
+  module Validators
+    class StringValidator < BaseValidator
+      # Coerce given value to the type the validator is validating against.
+      # PLEASE NOTE: no checks are performed on the given $value.
+      # It is therefore recommended to call the "run" method first before this.
+      #
+      # @param value mixed The value to coerce.
+      # @return mixed The same value.
+      def coerce(value)
+        value
```

Comment:
Not super sure on Ruby syntax, but is it worth returning `value.to_s` if it responds to it?

Reply:
This did originally do that, but it causes problems for cases where the types array allows both a model instance and a string.

i.e. identifier in the Lease model

```
      # @return int|string|OpenActive::Models::PropertyValue|OpenActive::Models::PropertyValue[]|null
      attr_accessor :identifier
      property :identifier, as: "identifier"
      validate_property :identifier, types: [
        "int",
        "string",
        "OpenActive::Models::PropertyValue",
        "OpenActive::Models::PropertyValue[]",
        "null",
      ]
```

In that case, it'll hit the string rule first. 

```
[21] pry(main)> OpenActive::Models::PropertyValue.new(value: 'abc')
=> #<OpenActive::Models::PropertyValue:0x00007ff1c641bdf0 @value="abc">
[22] pry(main)> OpenActive::Models::PropertyValue.new(value: 'abc').to_s
=> "#<OpenActive::Models::PropertyValue:0x00007ff1c5df10d8>"
```

Reply:
ah, .to_str is safer here.

```
[23] pry(main)> OpenActive::Models::PropertyValue.new(value: 'abc').to_str
NoMethodError: undefined method `to_str' for #<OpenActive::Models::PropertyValue:0x00007ff1c5e11540>
Did you mean?  to_s
from (pry):21:in `__pry__'

[24] pry(main)> "abc".to_str
=> "abc"
```

Reply:
Changed.